source/crud/crud.rst

```diff
@@ -672,7 +672,8 @@ As documented above, the only supported option is maxTimeMS::
   ]
 
 Similar to the count command, the estimated count of documents is returned
-in the ``n`` field.
+in the ``n`` field. Drivers can assume that the first field of the first batch
```

Comment:
This language is a bit confusing, as it's not the first field of the first batch.  How about something like:

Implementations can assume that the document containing the single result of the aggregation pipeline is contained in the first batch in the server's reply to the aggregate command.  It is not necessary to execute a getMore operation to ensure that the result is available.

Reply:
Yes, that sounds much better. Used almost exactly what you wrote.